scripts/build_components.sh

```diff
@@ -73,12 +74,17 @@ for dir in "${components_to_build[@]}"; do
     fi
     echo "$full_image_name"
     full_image_names+=("$full_image_name")
+    aws_full_image_names+=("$aws_full_image_name")
   done
 
   echo "Updating the image version in the fondant_component.yaml with:"
   echo "${full_image_names[0]}"
   sed -i -e "s|^image: .*|image: ${full_image_names[0]}|" fondant_component.yaml
 
+  # create repo if not exists
+  aws ecr-public describe-repositories --region us-east-1 --repository-names ${BASENAME} || aws ecr-public create-repository --region us-east-1 --repository-name ${BASENAME}
+  full_image_names+=("public.ecr.aws/m4f5a0g0/${BASENAME}:${tag}")
```

Comment:
This is temporary until we get the `fndnt` name reserved.

Reply:
Can you update this now?